H5P/laravel-h5p/src/LaravelH5p/LaravelH5p.php

```diff
@@ -223,8 +223,8 @@ public function get_embed($content, $settings, $no_cache = FALSE)
     private static function get_core_settings()
     {
         $contentUserDataUrl = config('app.url') . '/api/v1/h5p/ajax/content-user-data' . '?content_id=:contentId&data_type=:dataType&sub_content_id=:subContentId';
-        if (isset($_GET['gcuid'])) {
-            $contentUserDataUrl = config('app.url') . '/api/v1/google-classroom/h5p/ajax/content-user-data' . '?content_id=:contentId&data_type=:dataType&sub_content_id=:subContentId'.'&gcuid='.$_GET['gcuid'];
+        if (isset($_GET['gcuid']) && isset($_GET['submissionid'])) {
+            $contentUserDataUrl = config('app.url') . '/api/v1/google-classroom/h5p/ajax/content-user-data' . '?content_id=:contentId&data_type=:dataType&sub_content_id=:subContentId'.'&gcuid='.$_GET['gcuid'].'&submissionid='.$_GET['submissionid'];
```

Comment:
Please Add before and after concatenation operator and also split code into other line to avoid scroling